src/ui/page/share/dataset/steps/summary/summary.tsx

```diff
@@ -94,6 +100,31 @@ export const Summary: FC = () => {
 
         return null
       }
+
+      case 'date-range': {
+        const dateRangeValue = getDateRangeFieldValue(item)
+
+        if (!dateRangeValue) return null
+
+        const { from, to } = dateRangeValue
+
+        if (!from && !to) return null
+
+        if (from && to) {
+          return (
+            <p className="okp4-dataverse-portal-share-dataset-summary-item-text">
+              {`${t('from')} ${from} ${t('to').toLowerCase()} ${to}`}
+            </p>
+          )
+        }
+
+        return (
```

Comment:
there are some code duplication but I think it's more readable as you made, but I think you could maybe just refactor a bit:

```
        const dateRangeValue = getDateRangeFieldValue(item)

        if (!dateRangeValue || (!dateRangeValue.from && !dateRangeValue.to)) return null

        const { from, to } = dateRangeValue

        return from && to ? (
          <p className="okp4-dataverse-portal-share-dataset-summary-item-text">
            {`${t('from')} ${from} ${t('to').toLowerCase()} ${to}`}
          </p>
        ) : (
          <p className="okp4-dataverse-portal-share-dataset-summary-item-text">
            {from ? `${t('from')} ${from}` : `${t('to')} ${to}`}
          </p>
        )
```
We can continue to refactor to keep only one `<p>` but it's how you prefer.

Reply:
Thanks for the suggestion, it's cleaner with grouped null checks!